lib/blocs/groups_cubit.dart

```diff
@@ -1,8 +1,12 @@
+import 'dart:async';
+
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:reaxit/api/api_repository.dart';
 import 'package:reaxit/api/exceptions.dart';
-import 'package:reaxit/blocs/detail_state.dart';
 import 'package:reaxit/models/group.dart';
+import 'package:reaxit/config.dart' as config;
+
+import 'detail_state.dart';
```

Comment:
Could you make this a package import (package:reaxit/...) or perhaps just import .../models.dart)